builder/bdwgc.go

```diff
@@ -59,7 +65,6 @@ var BoehmGC = Library{
 			"blacklst.c",
 			"dbg_mlc.c",
```

Comment:
With the latest bdwgc master: `dbg_mlc.c` is not needed unless you use `GC_debug_*` API.